virt/logging_events_monitoring/virt-virtualization-alerts.adoc

```diff
@@ -0,0 +1,16 @@
+[id="virt-virtualization-alerts"]
+= OpenShift Virtualization alerts
+include::modules/virt-document-attributes.adoc[]
+include::modules/common-attributes.adoc[]
+:context: virt-virtualization-alerts
+
+toc::[]
+
+:FeatureName: OpenShift Virtualization alerts
+include::snippets/technology-preview.adoc[]
+
+OpenShift Virtualization uses alerts to provide additional information for its metrics. Each alert has a corresponding Operator.
```

Comment:
Each alert has a corresponding alert runbook.

Reply:
Would we tell downstream customers that each alert has an upstream runbook?